rest/config.go

```diff
@@ -138,13 +139,14 @@ func (bc *BucketConfig) MakeBucketSpec() base.BucketSpec {
 	}
 
 	return base.BucketSpec{
-		Server:     server,
-		BucketName: bucketName,
-		Keypath:    bc.KeyPath,
-		Certpath:   bc.CertPath,
-		CACertPath: bc.CACertPath,
-		KvTLSPort:  tlsPort,
-		Auth:       bc,
+		Server:                   server,
+		BucketName:               bucketName,
+		Keypath:                  bc.KeyPath,
+		Certpath:                 bc.CertPath,
+		CACertPath:               bc.CACertPath,
+		CACertUnsetTlsSkipVerify: dc.Unsupported.CACertUnsetTlsSkipVerify,
```

Comment:
Is there a valid scenario where users would want to set both CACertPath and the unsupported TlsSkipVerify?  If they are trying to use a self-signed cert they don't need to set CACertPath in the first place, I don't think.  Assuming there isn't a valid scenario, I feel like we should throw a config validation error if both are set.  As mentioned above, this allows us to just check Spec.TlsSkipVerify=true to see whether we're running with tlsSkipVerify, and not have to repeat the CACertPath check everywhere.